vmhost/hostCore/execution.go

```diff
@@ -957,6 +971,13 @@ func (host *vmHost) ExecuteESDTTransfer(transfersArgs *vmhost.ESDTTransfersArgs,
 		}
 	}
 
+	if len(transfersArgs.Function) > 0 {
+		esdtTransferInput.Arguments = append(esdtTransferInput.Arguments, []byte(transfersArgs.Function))
+	}
+	if len(transfersArgs.Arguments) > 0 {
+		esdtTransferInput.Arguments = append(esdtTransferInput.Arguments, transfersArgs.Arguments...)
```

Comment:
Is this ok? What happens if the esdtTransferInput struct contains the Function field set to empty and it contains a list of arguments? Should we prevent that situation? If yes, we need to move L977-L979 after L975

Reply:
that is not possible.